certif/app/models/user.js

```diff
@@ -0,0 +1,9 @@
+import Model, { attr } from '@ember-data/model';
```

Comment:
Vu avec l'équipe certif, on réintroduit le modèle User pour la création de compte Pix uniquement. Le modèle certification-point-of-contact reste le modèle prévu pour définir un utilisateur Pix Certif.

Autre proposition évoquée : nouvelle route API /certification-point-of-contact pour créer un utilisateur Pix Certif et ne pas réintroduire le User dans cette application.

> Probablement intéressant pour Pix Orga aussi avec sa notion de Prescriber.

Reply:
Vu avec l'équipe accès, ok pour le modèle user.